WooCommerce/Classes/ViewRelated/Blaze/CampaignCreation/BlazeCampaignCreationFormViewModel.swift

```diff
@@ -119,6 +119,13 @@ final class BlazeCampaignCreationFormViewModel: ObservableObject {
     @Published private(set) var errorState: ErrorState = .none
     private var suggestions: [BlazeAISuggestion] = []
 
+    var canConfirmDetails: Bool {
+        guard !isDownloadingImage else {
+            return false
+        }
+        return tagline.isNotEmpty && description.isNotEmpty
```

Comment:
I'm trying to understand what's the likelihood of the tagline/description to be empty, in BlazeCampaignCreationFormView.

If I understand correctly, this can only happen when entering the form, while `loadAISuggestions()` has not populated them. But when that is happening, the Text view is showing "Placeholder tagline", while the underlying data is empty.

From the user POV, though, they see a value there ("Placeholder tagline"), so it might be odd that the button is disabled at this point.

Is the placeholder tagline supposed to be updated / worked on later on? Maybe with a shimmer instead?

Reply:
> From the user POV, they see a value there ("Placeholder tagline"), so it might be odd that the button is disabled.
> Is the placeholder tagline supposed to be updated / worked on later on? Maybe with a shimmer instead?

1. The placeholder strings are used for the redacted effect view to occupy some space. If the string is empty the redacted effect view will not be visible. 
2. Users won't be seeing the placeholder strings as the view will be redacted while loading. 


I added a delay to the stubbed AI suggestions response to show how the UI will look like. What do you think? 

https://github.com/woocommerce/woocommerce-ios/assets/524475/9aa940b5-c787-43fd-b151-c1410f510eba